docs/docs/sample-scripts/flow/search-parent-flows/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+tags:  
+  - flows
+  - reports
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Find all parent flows that invoke the specified child flow
+
+Author: [Michał Kornet](https://github.com/mkm17)
+
+This script 
+
+<Tabs>
+  <TabItem value="PowerShell">
+
+  ```powershell
+    $m365Status = m365 status --output text
+    if ($m365Status -eq "Logged Out") {
+        m365 login
+    }
```

Comment:
just a small tip we could use the `m365 login --ensure` to do the same what we have here in a single line